packit_service/service/api/runs.py

```diff
@@ -64,6 +69,11 @@ def process_runs(runs):
                         "status": row.status,
                     }
                 )
+                if model_type == "test_run" and not srpm_build:
```

Comment:
oh you got it already… I would probably go with the condition above, so it's more generic and it would ideally catch the first build/test of the pipeline, might be helpful for SRPMs in Copr.

Reply:
thanks for looking into it, I changed the condition to be more generic, PTAL